src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -104,20 +120,51 @@ class ThemeColorsEditor extends React.Component {
         </div>
         <div className="theme-colors-editor__picker">
           {selectedColor && (
-            <SketchPicker
-              color={colors[selectedColor]}
-              width="270px"
-              disableAlpha={!colorsWithAlpha.includes(selectedColor)}
-              onChangeComplete={nextColor =>
-                this.handleColorChange(selectedColor, nextColor)
+            <div>
+              <SketchPicker
+                color={colors[selectedColor]}
+                width="270px"
+                disableAlpha={!colorsWithAlpha.includes(selectedColor)}
+                onChangeComplete={nextColor =>
+                  this.handleColorChange(selectedColor, nextColor)
+                }
+                presetColors={uniqueColorArray}
+              />
+              {advancedColors &&
+              <div className="use-default"
```

Comment:
Why is this a div with an embedded span? It almost seems like you tried to implement the appearance from the ASCII sketch at https://github.com/mozilla/FirefoxColor/pull/929#discussion_r424025891 . Is that the case? If so, let me clarify: The `[ ]` in the sketch should not be interpreted as a box (i.e. what you've put in the `<span/>` below). Rather, it's used to illustrate the behavior of standard UI elements (checkbox).

The curent HTML/CSS has the following problems:
- Due to the use of `div` (without tabindex), the element is not focusable and lacks any visual cues that the element would do anything (`:hover`).
- There is no obvious visual indication that the default color is used, or that the color picker is in use. One potential way to resolve that is to make the "Use default color" a label of a checkbox, and toggle the visibility of the color picker upon (un)checking it. There may be other ways.

Reply:
I have a radio proposal
![radio](https://user-images.githubusercontent.com/922368/81861958-aaf13900-9571-11ea-8d3d-10941814989b.gif)
And a checkbox proposal.
![checkbox](https://user-images.githubusercontent.com/922368/81861985-b3497400-9571-11ea-8b47-bb32c829d8d0.gif)
In both I have the same issue: what do I do when the user clicks "other" or unselects the "use default"? Currently it does nothing.

For the checkbox proposal I have the following issue: I don't know how to disable the color picker, the "visual cue that it is not in use".

Do you prefer one of these? I have separated commits that you can check out and try.

https://github.com/andreicristianpetcu/FirefoxColor/commit/0c8b59cf660c0f87a2aa1cd2317826e362f81027 and https://github.com/andreicristianpetcu/FirefoxColor/commit/a34db2755f442f01b14e7e612b0b016f18ca42ce